lib/elastic_apm/spies/sqs.rb

```diff
@@ -56,6 +56,7 @@ def self.span_context(queue_name, region)
             service: { resource: "#{SUBTYPE}/#{queue_name}" },
             cloud: { region: region }
           }
+          # span links added here?
```

Comment:
@estolfo Is this logically where I should be adding the span links? I assume I'll need to grab the traceparent details from the initiating request

Reply:
I am not sure off the top of my head and would need to spend some time to investigate to give an answer. This is what I would suggest to investigate:
1. Test SQS by using localstack and step through the instrumentation to learn about where spans are created
2. Read the Span link spec in the apm repo
3. Asking the authors of the spec if you need any clarification
4. I believe span links are relevant when a batch of messages is processed. So I would set up a test scenario processing a batch of messages and find the point at which you can create a link between the spans
5. Look at the instrumentation in other agents. I know the Python agent has implemented this, for example